lib/tasks_private/spec_helper.rake

```diff
@@ -0,0 +1,157 @@
+namespace :manageiq do
+  namespace :providers do
+    namespace :ansible_tower do
```

Comment:
Yes, I agree. I copied this namespace from the existing rake file. Will update it